### PR TITLE
[MRG] switch to using GitHub actions

### PR DESCRIPTION
**Pull request by ctb**
Remove TravisCI config, add github actions config.

Merge https://github.com/dib-lab/screed/pull/84 and https://github.com/dib-lab/screed/pull/85 first.

- [ ] Is it mergeable?
- [ ] `make test` Did it pass the tests?
- [ ] `make clean diff-cover` If it introduces new functionality, is it tested?
- [ ] `make format diff_pylint_report doc` Is it well formatted?
- [ ] For substantial changes or changes to the command-line interface, is it
  documented in `CHANGELOG.md`? See [keepachangelog](http://keepachangelog.com/)
  for more details.
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by ctb** [comment]
ready for review @luizirber

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
-screed -- short read sequence utils in Python.
+# screed -- short read sequence utils in Python.
 
-The official central repository for screed is:
+[![Documentation](https://readthedocs.org/projects/screed/badge/?version=latest)](http://screed.readthedocs.io/en/latest/)
+<a href="https://pypi.org/project/screed/"><img alt="PyPI" src="https://badge.fury.io/py/screed.svg"></a>
+<a href="https://github.com/dib-lab/screed/blob/latest/doc/LICENSE.rst"><img alt="License: 3-Clause BSD" src="https://img.shields.io/badge/License-BSD%203--Clause-blue.svg"></a>
+![Python tests](https://github.com/dib-lab/sourmash/workflows/Python%20tests/badge.svg)
```
Comment (luizirber) [inline README.md]: not sourmash =]
```suggestion
![Python tests](https://github.com/dib-lab/screed/workflows/Python%20tests/badge.svg)
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+
+The official repository for screed is:
 
    https://github.com/dib-lab/screed
 
 See http://readthedocs.org/docs/screed/en/latest/ for docs.
 
 Issues are tracked at https://github.com/dib-lab/khmer/issues.
-
-[![Build Status](http://ci.ged.msu.edu/job/screed/badge/icon)](http://ci.ged.msu.edu/job/screed/)
```
Comment (luizirber) [inline README.md:11]: oh my, was this outdated or what...

Comment (ctb) [reply README.md:11]: 👍

**Review by luizirber — APPROVED** [review]

**Pull request merged**